DiscordBot/index.js

```diff
@@ -1,22 +1,39 @@
 const Discord = require('discord.js');
+const commando = require('discord.js-commando');
 const login = require('./loginCode');
-const bot = new Discord.Client();
+const client = new commando.Client({
+    commandPrefix: '?'
+});
+
+client.registry
+.registerGroups([
+    ['random', 'Random'],
+    ['mud', 'MUD']
+])
+.registerDefaults()
+.registerCommandsIn(__dirname + "/commands");
 
-bot.login(login.key);
+client.login(login.key);
 
-bot.on('message', (message) => {
+console.log("Bot is running");
+
+// below this line are the pre-existing methods that the bot used to parse
+// messages
+client.on('message', (message) => {
 	if (message.content == 'ping') {
 		message.channel.send('pong');
 	}
+
+	let roleID = "530096793422266428";
```

Comment:
all these hard coded roles and rooms should be sanitized and automated/changed before merge imo. Get a second opinion I guess.

Reply:
Roles can only be set up whenever we receive a message as they use the message object itself to reference the guild which then has all of the roles. Also see my comment on the move.js file.